api/environments/identities/views.py

```diff
@@ -21,6 +30,62 @@
 from util.views import SDKAPIView
 
 
+class EdgeIdentityViewSet(viewsets.ModelViewSet):
+    serializer_class = EdgeIdentitySerializer
+    permission_classes = [IsAuthenticated, NestedEnvironmentPermissions]
+    pagination_class = EdgeIdentityPagination
+    lookup_field = "identity_uuid"
+
+    @staticmethod
+    def _get_search_function_and_value(
+        search_query: str,
+    ) -> typing.Tuple[typing.Callable, str]:
+        if search_query.startswith('"') and search_query.endswith('"'):
+            return dynamo_identifier_search_functions["EQUAL"], search_query.replace(
+                '"', ""
+            )
+        else:
+            return dynamo_identifier_search_functions["BEGINS_WITH"], search_query
+
+    def get_object(self):
+        return Identity.dynamodb.get_item_from_uuid(
+            self.kwargs["environment_api_key"], self.kwargs["identity_uuid"]
+        )
+
+    def get_queryset(self):
+        page_size = self.pagination_class().get_page_size(self.request)
+        previous_last_evaluated_key = self.request.GET.get("last_evaluated_key")
+        search_query = self.request.query_params.get("q")
+        start_key = None
+        if previous_last_evaluated_key:
+            start_key = json.loads(base64.b64decode(previous_last_evaluated_key))
+
+        if not search_query:
+            return Identity.dynamodb.get_all_items(
+                self.kwargs["environment_api_key"], page_size, start_key
+            )
+        search_func, search_identifier = self._get_search_function_and_value(
+            search_query
+        )
+        identity_documents = Identity.dynamodb.search_items_with_identifier(
+            self.kwargs["environment_api_key"],
+            search_identifier,
+            search_func,
+            page_size,
+            start_key,
+        )
+        return identity_documents
+
+    def get_environment_from_request(self):
```

Comment:
Why do we need this?

Reply:
I guess we can inline this?

Reply:
But it's not even used is it?

Reply:
Oh it is being used here: https://github.com/Flagsmith/flagsmith/pull/530/files#diff-55e28274d29ce084198b7aacadcd41d63b764f7adea2d1be9d6b15f19da96907R41. It was not being used in the initial commit tho, so that's my bad